uint8   possessionType
uint8   possessionRobotID
char    possessionTeam

